src/pages/Login/Login.scss

```
@@ -0,0 +1,105 @@
+@mixin flex($justify: center, $align: center) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+.login-header {
+  min-width: 400px;
+  .login-nav {
+    @include flex;
+    border-bottom: 1px solid #f0f0f0;
+    h1 {
+      padding: 10px 170px;
+      font-size: 17px;
+      font-weight: 500;
+    }
+    button {
```

새로운 선택자 간에 한 줄씩 띄워주세요!
```suggestion
.login-header {
  min-width: 400px;
  
  .login-nav {
    @include flex;
    border-bottom: 1px solid #f0f0f0;
    
    h1 {
      padding: 10px 170px;
      font-size: 17px;
      font-weight: 500;
    }
    
    button {
```

---

리뷰하지 않은 다른 부분도 수정해주세요!